talent/onboarding/Non-EU_residents.md

```diff
@@ -1,128 +1,142 @@
+# Candidates from Outside the European Union
+
+This is the hardest process and we need approval from the Spanish Government. We have to get a work permit and a visa in the candidate's passport. The first goal, the work permit, is the responsibility of the Company and the process must take place in Spain. The second goal, the visa in the passport, is the responsibility of the candidate and must take place in the Spanish Consulate in the country where the candidate is living.
+
+
+## Work Permit
 
-## Work permit
 We will need the following documents:
 
 1. From the candidate side:
-    * Passport (every page)
-    * CV - Resume -- in Spanish
-    * Reference letters -- in Spanish
-    * Diploma - Studies certificate -- in Spanish. (Sworn translation). Sometimes we need the diploma with the Haya Apostille. 
+
+    * Passport: All pages.
+    * CV/Resume: in Spanish.
+    * Reference letters: in Spanish.
+    * Diploma/Studies certificate: in Spanish (sworn translation). Sometimes we need the diploma with the Haya Apostille.
 
 2. From the company side:
+
     * The same for every candidate
       - source{d} CIF.
       - Jorge’s ID
-      - Strategy area - certificate. 
-      - Power of attorney
-      - Resident authorization 
+      - Strategy area: Certificate.
+      - Power of attorney.
+      - Resident authorization.
+
     * Different for every candidate
-      - Paloma’s authorization. 
-      - Work contract- It's a draft.
-      - Business activity. 
+      - Paloma’s authorization.
+      - Work contract: Draft.
+      - Business activity.
       - Project memory.
-      - Job profile. 
+      - Job profile.
+
+When we have all the documents ready, the lawyer will present them to the government. The government has 20 business days to decide if the application is good or not. If so, the government gives us the “work permit” with a NIE number.
 
-When we have all the documents ready, the lawyer will present them to the government. The government has 20 labor days to decide if the application is good or not. The government is going to give us the “work permit” with a NIE number.  
 
+## Visa in the Passport
 
-## Visa in the passport
+Depending on the country of origin, the candidate may require different documentation:
 
-Depends on the country, you are going to need different docs:
-  1. Filled form w/ photo - The consulate will give you. 
-  2. Passport
-  3. ID from the original country.
-  4. Job Offer with salary indication
+  1. Filled application form with photo: The consulate provides this to the candidate.
+  2. Passport.
```

Comment:
Using `1.` for all list items will use sequential numbers, which is easier to maintain.
Not a requirement for this PR, just mentioning :dancing_men:

Reply:
Personally I dislike that, because the point of Markdown was that it's also readable as plain text, and a numbered list doesn't make sense if everything is "1.". I understand the motivation, but .md is primarily for humans, not computers IMO.